src/app/features/room/RoomInput.tsx

```diff
@@ -289,25 +288,41 @@ export const RoomInput = forwardRef<HTMLDivElement, RoomInputProps>(
 
       if (plainText === '') return;
 
-      let body = plainText;
-      let formattedBody = customHtml;
-      if (replyDraft) {
-        body = parseReplyBody(replyDraft.userId, trimReplyFromBody(replyDraft.body)) + body;
-        formattedBody =
-          parseReplyFormattedBody(
-            roomId,
-            replyDraft.userId,
-            replyDraft.eventId,
-            replyDraft.formattedBody
-              ? trimReplyFromFormattedBody(replyDraft.formattedBody)
-              : sanitizeText(replyDraft.body)
-          ) + formattedBody;
-      }
+      const body = plainText;
+      const formattedBody = customHtml;
 
       const content: IContent = {
         msgtype: msgType,
         body,
       };
+      const userIdMentions = new Set<string>();
+      if (replyDraft) {
+        userIdMentions.add(replyDraft.userId);
```

Comment:
this adds users own userId if they are replying to own message, not sure if this is intentional?